### PR TITLE
2to3, workaround breaking changes

### PR DESCRIPTION
**Pull request by scienceopen**
Not everyone needs to install pyspeckit.

### CODE REVIEW

File: gaussfitter/mpfit/mpfit.py [hunk 1]
```diff
@@ -407,11 +407,16 @@ def myfunct(p, fjac=None, x=None, y=None, err=None)
...
 import numpy
-import types
-from pyspeckit.spectrum.parinfo import ParinfoList,Parinfo
+from six import integer_types
 from astropy import log
+try:
+    from pyspeckit.spectrum.parinfo import ParinfoList,Parinfo
+    haspyspeckit = True
+except:
+    haspyspeckit = False
```
Comment (keflavich) [inline gaussfitter/mpfit/mpfit.py]: can we make this `HASPYSPECKIT` or `PYSPECKITOK`?  I think there is a convention that global variables like this should be all caps, though I can't point to a specific doc on it...

File: gaussfitter/mpfit/mpfit.py [hunk 2]
```diff
@@ -872,15 +877,16 @@ def iterfunct(myfunct, p, iter, fnorm, functkw=None,
 
         # Be sure that PARINFO is of the right type
         if parinfo is not None:
-            if type(parinfo) is ParinfoList:
+            if haspyspeckit and isinstance(parinfo,ParinfoList):
```
Comment (keflavich) [inline gaussfitter/mpfit/mpfit.py]: :+1:, this was from before I knew how to use `isinstance` properly.

**Comment by keflavich** [comment]
Looks good.  Did you want to do `astropy.log` -> `logging.log` before I merge this?

**Comment by scienceopen** [comment]
OK I made the changes:
1) Just use standard library logging
2) capitalize HAS_PYSPECKIT variable

**Pull request merged**